src/algorithms/synthesis/encoding.cpp

```diff
@@ -131,28 +142,26 @@ namespace syrec {
             output = encoding[output];
         }
 
-        return {encoding, additionalLines};
+        return encoding;
     }
 
-    auto augmentWithConstants(TruthTable& tt, std::size_t const& nBits, bool dc, bool appendZeroOut, bool appendZeroIn) -> void {
+    auto augmentWithConstants(TruthTable& tt, std::size_t const& nBits, bool appendZero) -> void {
         const auto requiredOutConstants = nBits - tt.nOutputs();
         const auto requiredInConstants  = nBits - tt.nInputs();
```

Comment:
This function should appropriately set the `constants` (and possibly the garbage) vector of the TT.